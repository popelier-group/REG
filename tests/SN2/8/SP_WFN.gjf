# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.55306600   -0.00006300    0.00021600
 H                 -0.80515800   -0.88966000   -0.55399700
 H                 -0.80520400    0.92470400   -0.49304900
 H                 -0.80503700   -0.03525800    1.04776900
 Br                 1.58978200    0.00000200   -0.00000900
 Cl                -3.32844300    0.00002400   -0.00007600

frame.wfn
