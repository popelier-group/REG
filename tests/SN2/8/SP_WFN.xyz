
6

C       -0.438628    0.009938    0.065754
H       -0.568471   -0.541528    0.982875
H       -0.835768   -0.474642   -0.811480
H       -0.659320    1.062644    0.137870
Br        1.680061   -0.037281   -0.251620
Cl       -3.182752    0.070513    0.476642
