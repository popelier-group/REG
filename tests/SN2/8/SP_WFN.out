 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.8/Gau-111182.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.8/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=    111183.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.55307  -0.00006   0.00022 
 H                    -0.80516  -0.88966  -0.554 
 H                    -0.8052    0.9247   -0.49305 
 H                    -0.80504  -0.03526   1.04777 
 Br                    1.58978   0.       -0.00001 
 Cl                   -3.32844   0.00002  -0.00008 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.553066   -0.000063    0.000216
      2          1           0       -0.805158   -0.889660   -0.553997
      3          1           0       -0.805204    0.924704   -0.493049
      4          1           0       -0.805037   -0.035258    1.047769
      5         35           0        1.589782    0.000002   -0.000009
      6         17           0       -3.328443    0.000024   -0.000076
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.078001   0.000000
     3  H    1.077997   1.815387   0.000000
     4  H    1.078005   1.815394   1.815392   0.000000
     5  Br   2.142848   2.614219   2.614215   2.614238   0.000000
     6  Cl   2.775377   2.732276   2.732177   2.732545   4.918225
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.438628    0.009938    0.065754
      2          1           0       -0.568471   -0.541528    0.982875
      3          1           0       -0.835768   -0.474642   -0.811480
      4          1           0       -0.659320    1.062644    0.137870
      5         35           0        1.680061   -0.037281   -0.251620
      6         17           0       -3.182752    0.070513    0.476642
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         152.1568395           0.8505865           0.8505865
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       176.1689937758 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  8.55D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.39255416     A.U. after   11 cycles
            NFock= 11  Conv=0.89D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.67034-101.26945 -62.32329 -56.14294 -56.14007
 Alpha  occ. eigenvalues --  -56.14007 -10.08068  -9.18279  -8.53240  -6.94182
 Alpha  occ. eigenvalues --   -6.94065  -6.94065  -6.36143  -6.35203  -6.35203
 Alpha  occ. eigenvalues --   -2.47266  -2.47005  -2.47005  -2.46260  -2.46260
 Alpha  occ. eigenvalues --   -0.63599  -0.54366  -0.51571  -0.29886  -0.29886
 Alpha  occ. eigenvalues --   -0.21783  -0.14411  -0.14411  -0.06898  -0.06834
 Alpha  occ. eigenvalues --   -0.06834
 Alpha virt. eigenvalues --    0.09807   0.11177   0.14495   0.14495   0.15795
 Alpha virt. eigenvalues --    0.17606   0.18143   0.18143   0.21121   0.22002
 Alpha virt. eigenvalues --    0.22003   0.24093   0.24093   0.24799   0.31514
 Alpha virt. eigenvalues --    0.31514   0.31583   0.34949   0.40277   0.50893
 Alpha virt. eigenvalues --    0.53256   0.53256   0.54746   0.54746   0.65399
 Alpha virt. eigenvalues --    0.66965   0.66966   0.69793   0.75370   0.75370
 Alpha virt. eigenvalues --    0.75475   0.78602   0.78602   0.83498   0.83499
 Alpha virt. eigenvalues --    0.84855   0.93956   1.09661   1.13661   1.13661
 Alpha virt. eigenvalues --    1.14144   1.14145   1.15697   1.22978   1.22979
 Alpha virt. eigenvalues --    1.30048   1.55513   1.66287   1.66287   1.77133
 Alpha virt. eigenvalues --    1.87849   1.87849   2.04927   2.13357   2.13357
 Alpha virt. eigenvalues --    2.28397   2.28397   2.29569   2.51478   2.51479
 Alpha virt. eigenvalues --    2.57790   2.62486   2.64046   2.64046   2.71727
 Alpha virt. eigenvalues --    2.71727   3.08048   3.19454   3.85525   3.85526
 Alpha virt. eigenvalues --    4.29403   4.29403   4.36573   4.36573   4.62589
 Alpha virt. eigenvalues --    6.73715   7.70672   7.70673   7.81363   9.88145
 Alpha virt. eigenvalues --   24.04404  26.07648  26.07650  26.24575  48.03429
 Alpha virt. eigenvalues --  215.85176 289.90709 289.90709 290.022981020.81024
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.056760   0.414127   0.414134   0.414104   0.082400  -0.063423
     2  H    0.414127   0.525481  -0.034182  -0.034179  -0.051898   0.000289
     3  H    0.414134  -0.034182   0.525470  -0.034180  -0.051898   0.000288
     4  H    0.414104  -0.034179  -0.034180   0.525511  -0.051892   0.000293
     5  Br   0.082400  -0.051898  -0.051898  -0.051892  35.456846   0.007851
     6  Cl  -0.063423   0.000289   0.000288   0.000293   0.007851  17.886262
 Mulliken charges:
               1
     1  C   -0.318102
     2  H    0.180361
     3  H    0.180367
     4  H    0.180343
     5  Br  -0.391408
     6  Cl  -0.831560
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.222968
     5  Br  -0.391408
     6  Cl  -0.831560
 Electronic spatial extent (au):  <R**2>=           1145.9457
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              8.8773    Y=             -0.1963    Z=             -1.3293  Tot=              8.9784
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -92.6806   YY=            -43.8029   ZZ=            -44.8756
   XY=              1.0834   XZ=              7.3234   YZ=             -0.1622
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -32.2276   YY=             16.6501   ZZ=             15.5774
   XY=              1.0834   XZ=              7.3234   YZ=             -0.1622
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=            199.6656  YYY=             -0.4579  ZZZ=             -9.9606  XYY=             21.0460
  XXY=             -3.5167  XXZ=            -23.6377  XZZ=             23.9778  YZZ=             -1.4490
  YYZ=             -3.0467  XYZ=              0.3191
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1863.6505 YYYY=            -70.6874 ZZZZ=           -101.2721 XXXY=             24.4894
 XXXZ=            165.2478 YYYX=             14.6911 YYYZ=             -2.1954 ZZZX=            104.8147
 ZZZY=             -2.5947 XXYY=           -253.8116 XXZZ=           -267.8286 YYZZ=            -28.6406
 XXYZ=             -1.9584 YYXZ=             34.4234 ZZXY=              5.8815
 N-N= 1.761689937758D+02 E-N=-7.704093027578D+03  KE= 3.069985629961D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE539\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.553066,-0.000063,0.000216\H,0,-0.805158,-0.88966,-0.553997\H,0,
 -0.805204,0.924704,-0.493049\H,0,-0.805037,-0.035258,1.047769\Br,0,1.5
 89782,0.000002,-0.000009\Cl,0,-3.328443,0.000024,-0.000076\\Version=ES
 64L-G16RevC.01\State=1-A\HF=-3074.3925542\RMSD=8.917e-09\Dipole=3.5323
 728,-0.0000669,0.0002294\Quadrupole=-24.7936543,12.3968445,12.3968098,
 0.0002893,-0.0009545,-0.0000041\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 KNOWLEDGE IS OF TWO KINDS:  WE KNOW A SUBJECT OURSELVES OR WE KNOW 
 WHERE WE CAN FIND INFORMATION UPON IT.

                                    -- SAMUEL JOHNSON
 Job cpu time:       0 days  0 hours  0 minutes 29.1 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 15.4 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:08 2020.
